ssas/service/admin/router.go

```diff
@@ -15,6 +15,7 @@ func init() {
 func Routes() *chi.Mux {
 	r := chi.NewRouter()
 	r.Post("/group", createGroup)
+	r.Put("/group/{id}", updateGroup)
```

Comment:
Line 12 just above this (can't seem to tag it . . .) should somehow be updated to include the new endpoint.

Reply:
my understanding (from what we have already merged) was that the `/_info` endpoint just shows the resources that have methods in the admin namespace.  do you still think there is something to change here?

Reply:
Yes, nothing to add without a refactor.  Probably fine for now.